### PR TITLE
Signup

### PR DESCRIPTION
**Pull request by topcheese044**
Fix: Error handling is done correctly. Check if invalid emails or duplicate emails work.

### CODE REVIEW

**Comment by topcheese044** [comment]
> Definitely needs work. Duplicate emails are seen as valid, and I can register the same email several times.
> On another note, there should be a password requirement (at least 8 characters, one number, one capital, one symbol)
> 
> image

File: store/auth.ts [hunk 1]
```diff
@@ -85,13 +85,35 @@ export default class AuthModule extends VuexModule implements AuthState {
...
+        {
+            if (error.code === 'auth/email-already-in-use') {
+                console.log('in use')
+            }
+        
+            else if (error.code === 'auth/invalid-email') {
+                console.log('invalid email')
+            }
+            console.log(error);
         }
```
[the remark this answers is not in the document]
Comment (topcheese044) [reply store/auth.ts:117]: is it not thrown at the bottom?

[the remark this answers is not in the document]
Comment (DrCheddar) [reply store/auth.ts:117]: what line?

[the remark this answers is not in the document]
Comment (DrCheddar) [reply store/auth.ts:117]: what line?

**Pull request closed without merging**